x-pack/plugins/apm/public/components/app/RumDashboard/translations.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const EndUserExperienceLabel = i18n.translate(
+  'xpack.apm.rum.dashboard.title',
+  {
+    defaultMessage: 'End User Experience',
+  }
+);
+
+export const getWhatIsGoingOnLabel = (environmentVal: string) =>
+  i18n.translate('xpack.apm.rum.dashboard.environment.title', {
+    defaultMessage: `What's going on in {environmentVal}?`,
```

Comment:
This seems a little more casual than the other titles we have. I actually kinda like it but I think it stands out. @formgeist/ @bmorelli25  WDYT?

For context:
![image](https://user-images.githubusercontent.com/209966/84675407-7edc2580-af2c-11ea-89b2-e5a859e77eb2.png)

Reply:
... and when I select an environment

![image](https://user-images.githubusercontent.com/209966/84675497-9fa47b00-af2c-11ea-92ec-29881ada32f0.png)

Reply:
Is this a bug? (undefined environment)

![image](https://user-images.githubusercontent.com/209966/84675540-b054f100-af2c-11ea-8e89-b83b224b1e9a.png)

Reply:
> "What's going on..." seems a little more casual than the other titles we have. I actually kinda like it but I think it stands out.

Do we need a title on this tab? We don't have one on the other tabs, and I'm not sure that it adds anything here.

Reply:
i think this tabs is different in a perspective that it provides an overview, i personally would like to keep the title even though i got it from @katrin-freihofner mock. It immediately draws your attention to the page and gives you a clear cut meaning what this page supposed to mean.

Reply:
I guess the problem I have is that "What's going on in _environment_" could be the title of any chart in Kibana. It doesn't tell us anything unique about what we're seeing on the page. Whereas something like "analytics for _env_", "recent activity in your _env_,", "loading trends",  or "user experience" is specific and describes what the collection of data is displaying.

It'd also be nice if this title matched the formatting of the other APM tabs, which looks to be smaller and bold.

<img width="892" alt="Screen Shot 2020-06-15 at 12 53 16 PM" src="https://user-images.githubusercontent.com/5618806/84700012-5b52b400-af07-11ea-9bea-ff5dafefa38f.png">